app/src/main/res/layout-land/administrator_controls_account_actions_view.xml

```diff
@@ -23,7 +23,7 @@
       android:paddingEnd="36dp"
       android:paddingBottom="20dp"
       android:text="@string/administrator_controls_account_actions_label"
-      android:textColor="@color/oppia_primary"
+      android:textColor="@color/primary_color"
```

Comment:
This color also does not add context.  Something like `sub_heading_color` might be more clear or `adminsitrator_controls_sub_heading_color`

Reply:
Changed to `admin_controls_sub_heading_color`.